eng/config/PublishData.json

```
@@ -27,14 +27,23 @@
           "vsBranch": "rel/d16.1",
           "vsMajorVersion": 16
         },
-        "master-vs-deps": {
+        "dev16.1-vs-deps": {
             "nugetKind": ["Shipping", "NonShipping"],
             "version": "3.1.*",
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev16.1", "dev16.1p3" ],
+            "channels": [ "dev16.1" ],
             "vsBranch": "lab/d16.1stg",
             "vsMajorVersion": 16
+          },
+        "master-vs-deps": {
+            "nugetKind": ["Shipping", "NonShipping"],
+            "version": "3.2.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev16.2p1" ],
```

Note the `p1`

---

I think this should also include 16.2.

```suggestion
            "channels": [ "dev16.2", "dev16.2p1" ],
```

---

There should also be a dev16.2 entry as well: we're saying this branch is simultaneously targeting preview1 and also it's the target for anything 16.2 in general. It's a meaningless distinction _now_, but once you have a second branch it clarifies somebody who just wants to know "what's the 16.2 branch? I don't care about specific previews".